--- telldus-core/service/CMakeLists.txt.orig	2018-02-15 21:54:58.342017698 +0100
+++ telldus-core/service/CMakeLists.txt	2018-02-15 21:44:02.706653427 +0100
@@ -184,9 +184,9 @@
 	LIST(APPEND telldus-service_SRCS TellStick_ftd2xx.cpp )
 	LIST(APPEND telldus-service_LIBRARIES ${FTD2XX_LIBRARY})
 ELSE (FTDI_ENGINE STREQUAL "ftd2xx")
-	FIND_LIBRARY(FTDI_LIBRARY ftdi)
+	FIND_LIBRARY(FTDI_LIBRARY ftdi1)
 	INCLUDE(FindPkgConfig)
-	PKG_SEARCH_MODULE(FTDI libftdi)
+	PKG_SEARCH_MODULE(FTDI libftdi1)
 	INCLUDE_DIRECTORIES( ${FTDI_INCLUDEDIR} )
 	ADD_DEFINITIONS( -DLIBFTDI )
 	LIST(APPEND telldus-service_SRCS TellStick_libftdi.cpp )
